File: MPI/mpicheck/tests/functions/3.0/MPI_Status_f082f.f90

        
        program main
        use mpi
        
        TYPE(MPI_Status) var_0
       INTEGER var_1(10)
       INTEGER var_2
        call mpi_status_f082f(var_0, var_1, var_2)
       call pmpi_status_f082f(var_0, var_1, var_2)
        end program main
    

File: MPI/mpicheck/tests/functions/3.0/MPI_Status_f082f.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const MPI_F08_status *var_0;
    MPI_Fint *var_1;
    int ret;
    /* calls */
    ret = MPI_Status_f082f(var_0, var_1);
    ret = PMPI_Status_f082f(var_0, var_1);
    return 0;
}
